### PR TITLE
docs(component): fixed typo and diff line numbers

### PR DESCRIPTION
**Pull request by sarbbottam**
rentals-hbs

diff

### CODE REVIEW

**Comment by sarbbottam** [comment]
Quite a bit of white spaces got removed, please use [w=1(https://github.com/emberjs/guides/pull/1648/files?w=1) as the query param while removing to reduce the noise.

File: source/localizable/tutorial/simple-component.de-DE.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ moduleForComponent('rental-listing', 'Integration | Component | rental listing',
...
-    
+
     * A template that defines how it will look (`app/templates/components/rental-listing.hbs`)
     * A JavaScript source file (`app/components/rental-listing.js`) that defines how it will behave.
-    
+
     Our new `rental-listing` component will manage how a user sees and interacts with a rental.
-    To start, let's move the rental display details for a single rental from the `index.hbs` template into `rental-listing.hbs` and add the image field:
-    
+    To start, let's move the rental display details for a single rental from the `rentals.hbs` template into `rental-listing.hbs` and add the image field:
```
Comment (sarbbottam) [inline source/localizable/tutorial/simple-component.de-DE.md]: `index.js` is empty as this stage. Should be `rentals.js`

File: source/localizable/tutorial/simple-component.de-DE.md [hunk 2]
```diff
@@ -51,26 +51,26 @@ test('should toggle wide class on click', function(assert) { assert.expect(3); l
         <span>Number of bedrooms:</span> {{rental.bedrooms}}
       </div>
     </article>
-    
+
 
 In our `rentals.hbs` template, let's replace the old HTML markup within our `{{#each}}` loop with our new `rental-listing` component:
 
-```app/templates/rentals.hbs{+13,+14,-15,-16,-17,-18,-19,-20,-21,-22,-23,-24,-25,-26,-27,-28,-29,-30} 
+```app/templates/rentals.hbs{+12,+13,-14,-15,-16,-17,-18,-19,-20,-21,-22,-23,-24,-25,-26,-27,-28,-29}
```
Comment (sarbbottam) [inline source/localizable/tutorial/simple-component.de-DE.md]: Please see the attached image.

**Comment by acorncom** [comment]
@sarbbottam Could you send over changes just to `simple-component.md`?  The other files get synced automatically so we'll actually have problems if you make changes there ...

Also, if you can remove your whitespace changes, that'd make life simpler here ;-)

**Comment by sarbbottam** [comment]
@acorncom Thanks for the feedback. Could you take another look?

File: source/localizable/tutorial/simple-component.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ A component consists of two parts:
 * A JavaScript source file (`app/components/rental-listing.js`) that defines how it will behave.
 
 Our new `rental-listing` component will manage how a user sees and interacts with a rental.
-To start, let's move the rental display details for a single rental from the `index.hbs` template into `rental-listing.hbs` and add the image field:
+To start, let's move the rental display details for a single rental from the `rentals.hbs` template into `rental-listing.hbs` and add the image field:
```
Comment (sarbbottam) [inline source/localizable/tutorial/simple-component.md]: `index.hbs` is empty as this stage, should be `rentals.hbs`

File: source/localizable/tutorial/simple-component.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ To start, let's move the rental display details for a single rental from the `in
 In our `rentals.hbs` template, let's replace the old HTML markup within our `{{#each}}` loop
 with our new `rental-listing` component:
 
-```app/templates/rentals.hbs{+13,+14,-15,-16,-17,-18,-19,-20,-21,-22,-23,-24,-25,-26,-27,-28,-29,-30}
+```app/templates/rentals.hbs{+12,+13,-14,-15,-16,-17,-18,-19,-20,-21,-22,-23,-24,-25,-26,-27,-28,-29}
```
Comment (sarbbottam) [inline source/localizable/tutorial/simple-component.md]: Please refer the attached image.

**Comment by toddjordan** [comment]
@acorncom @sarbbottam this one is a duplicate of #1645

**Comment by toddjordan** [comment]
Thanks for your contribution @sarbbottam .  This looks good to go so I'm going to go ahead and merge

**Comment by sarbbottam** [comment]
No problem @toddjordan. I am new to ember and following this guide to ramp up. Thanks for maintaining the guide, it's so helpful to ember novices like me.

**Pull request merged**